includes/actions/class-amp-frontend-actions.php

```diff
@@ -0,0 +1,19 @@
+<?php
+// Callbacks for adding AMP-related things to the main theme
+
+require_once ( AMP__DIR__ . '/includes/actions/class-amp-actions.php' );
+
+class AMP_Frontend_Actions extends AMP_Actions {
+	
+	public static function register_frontend_actions_filter() {
```

Comment:
For consistency, we should name this `register` as well (or `register_hooks` if we want to be more descriptive)